<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interventions</class>
 <widget class="QDialog" name="interventions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>801</width>
     <height>451</height>
    </rect>
   </property>
   <property name="title">
    <string>Gestion Des Interventions</string>
   </property>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Ajouter Une Intervention</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
    <widget class="QComboBox" name="etat_main">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Aucun</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Réparation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Révision</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Etat</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="type_main">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="modifier_main">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>250</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_main">
     <property name="geometry">
      <rect>
       <x>132</x>
       <y>20</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_mach">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID de la machine</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouter_main">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>250</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="date_main">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="descr_main">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Supprimer Une Intervention</string>
    </property>
    <widget class="QLineEdit" name="id_main_sup">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimer_main">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>431</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>Afficher Les Interventions</string>
    </property>
    <widget class="QTableView" name="afficher_main">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>411</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="mail">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Envoyer un mail </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="chercher">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>390</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>390</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Chercher</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
